pkg/validator/validators_test.go

```diff
@@ -10,13 +9,204 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+func TestIngressBackendValidator(t *testing.T) {
+	assert := tassert.New(t)
```

Comment:
Let's move this inside the `t.Run()` body in the loop so the error messages have the subtest context (#3745).